src/structs/Transitions.jl

```diff
@@ -37,5 +90,22 @@ Scaling(fs::Tuple, ts) = Scaling(Scale(fs...), ts)
 Scaling(fs::Union{Object,Scale,Symbol}, ts::Real) = Scaling(fs, Scale(ts, ts))
 Scaling(fs::Union{Object,Scale,Symbol}, ts::Tuple) = Scaling(fs, Scale(ts...))
 
+"""
+    anim_scale
+
+Animate the scaling of the attached object (see [`act!`](@ref)).
+**Attention:** Scaling is always done from the current origin.
+
+# Example
+```julia
+Background(1:100, ground)
+obj = Object((args...) -> rect(O, 50, 50, :fill), Point(100, 0))
+act!(obj, Action(1:50, anim_scale(1.5)))
+```
+
+# Options
+- `anim_scale(ts)` scales from the current scale to `ts`.
+- `anim_scale(fs, ts)` scales from `fs` to `ts`.
```

Comment:
I get that `ts` and `fs` represent "to scale" and "from scale" but could you add a type definition of whether they accept Floats, Ints, Reals? Thanks!

Reply:
There are quite a lot of different ways like Symbol, Number and object

Reply:
Maybe it's clearer now. I didn't document the symbol there as it's more for internal usage.